tests/core/dom/range/manual/getclientrectswitheditor.md

```diff
@@ -0,0 +1,27 @@
+@bender-tags: 4.10.0, feature, range, selection, 1902, 1915, 1925, 1930, tableselection
+@bender-ckeditor-plugins: wysiwygarea, toolbar, codesnippet, image2, table, tableselection, sourcearea, placeholder
```

Comment:
It would be nice to have also second editor in this test, without `tableselection` plugin as native table selection is also broken.